app/views/patients/_form.html.slim

```diff
@@ -1,152 +1,151 @@
+    .callout
+      h4 SCI Hub
       .row
         .medium-6.columns
-          = f.input :non_va_facility_name, label: "Facility"
-          = f.input :non_va_facility_phone_number, label: "Phone Number"
+          = f.association :assigned_sci_hub,
+          collection: Domain::SciHub.cached_all,
+          label: "Assigned SCI Center"
+          / todo: REMOVE
+          / .medium-3.columns
+          /   = f.input :assigned_sci_hub_physician_first_name, label: "Physician first name"
+          / .medium-3.columns
+          /   = f.input :assigned_sci_hub_physician_last_name, label: "Physician last name"
         .medium-6.columns
-          = f.input :non_va_facility_fax_number, label: "Fax number"
-        .medium-3.columns
-          = f.input :non_va_facility_pcp_first_name, label: "Physician first name"
-        .medium-3.columns
-          = f.input :non_va_facility_pcp_last_name, label: "Physician last name"
-    .callout-blank
+          = f.association :preferred_sci_hub,
+          collection: Domain::SciHub.cached_all,
+          label: "Preferred SCI Center (if different)"
+          / todo: REMOVE
+          / .medium-3.columns
+          /   = f.input :preferred_sci_hub_physician_first_name, label: "Physician first name"
+          / .medium-3.columns
+          /   = f.input :preferred_sci_hub_physician_last_name, label: "Physician last name"
+    .callout
       h4 SCI/D Information
       .row
         .medium-4.columns
-          = f.association :sci_type,
-          collection: Domain::SciType.cached_all,
-          prompt: "Select one",
-          label: "SCI Type"
-        .medium-4.columns
           = f.association :scid_eligibility,
           collection: Domain::ScidEligibility.cached_all,
           prompt: "Select one",
           label: "SCI/D Eligibility"
-          / only if other is selected above
-        .medium-4.columns
-          / TODO conditional only if SCI is selected above
-          = f.input :scid_eligibility_other, label: "Other"
-        .medium-4.columns
-          = f.input :sci_arrival_date, as: :date_picker, label: "SCI Arrival Date"
-        .medium-4.columns
-          = f.association :scid_etiology,
-          collection: Domain::ScidEtiology.cached_all,
-          prompt: "Select one",
-          label: "SCI/D Etiology"
-        .medium-4.columns
-          = f.input :date_of_injury, as: :date_picker
-        .medium-4.columns
-          = f.input :is_on_active_duty, label: "Currently on Active Duty"
-        .medium-8.columns
-          label.control-label.radio Theater of service
-          = f.input_field :theater_of_service_id,
-          collection: Domain::TheaterOfService.cached_all,
-          as: :radio_buttons,
-          item_wrapper_class: 'radio-inline'
-    = f.simple_fields_for :asia_assessment do |asia_assessment|
-      = render 'shared/asia_fields', :f => asia_assessment
-      / end conditional only if SCI is selected above
+        #sciOther.hide
```

Comment:
nit: I think we're inconsistent in whether we use dashes or lowerStartingCamelCase for identifier names. Which do you prefer? We should normalize.